tests/components/rflink/test_light.py

```diff
@@ -297,6 +299,7 @@ def listener(event):
     await hass.services.async_call(
         DOMAIN, SERVICE_TURN_OFF, {ATTR_ENTITY_ID: DOMAIN + ".test"}
     )
+    await asyncio.sleep(0)
```

Comment:
Will this be needed in general?

Reply:
This is a test about interweaving requests. Very fragile :( if we block till done or call with `blocking=True`, it's already done before we get the 2nd call !